src/managers/input/Tablets.cpp

```diff
@@ -36,6 +36,12 @@ void CInputManager::newTabletTool(wlr_input_device* pDevice) {
             const auto EVENT = (wlr_tablet_tool_axis_event*)data;
             const auto PTAB  = (STablet*)owner;
 
+            // Get tablet active area in mm and calculate bounds
+            const auto ACTIVE_AREA_SIZE = g_pConfigManager->getDeviceVec(PTAB->name, "active_area_size", "input:tablet:active_area_size");
```

Comment:
Extreme performance kill. Move the variable setting to `setTabletConfigs` and store a copy inside STablet itself.

Using `getDevice*` outside `set*Config` is banned

Reply:
Now that you mention it, I felt I was doing something off. Thanks for the heads-up, that makes much more sense.